source/src/MonitorElementManager.cc

```diff
@@ -197,7 +190,7 @@ namespace dqm4hep {
     //-------------------------------------------------------------------------------------------------
     //-------------------------------------------------------------------------------------------------
 
-    StatusCode MonitorElementManager::createQualityTests(TiXmlElement *const pXmlElement) {
+    StatusCode MonitorElementManager::parseQualityTests(TiXmlElement *pXmlElement) {
```

Comment:
Why removing the constness of the xmlElement? Parsing the element should not modify it.